src/encoded/static/components/doc.js

```diff
@@ -447,3 +447,34 @@ globals.documentViews.caption.register(QCAttachmentCaption, 'QualityMetric');
 
 // Register document preview rendering components
 globals.documentViews.preview.register(QCAttachmentPreview, 'QualityMetric');
+
+
+// Display a list of document links within a characterization.
+export const CharacterizationDocuments = (props) => {
+    const docs = props.docs.filter(doc => !!doc);
+    return (
+        <dd>
+            {docs.map((doc) => {
+                if (doc && doc.attachment) {
+                    const attachmentHref = url.resolve(doc['@id'], doc.attachment.href);
+                    const docName = (doc.aliases && doc.aliases.length) ? doc.aliases[0] :
+                        ((doc.attachment && doc.attachment.download) ? doc.attachment.download : '');
+                    return (
+                        <div className="multi-dd dl-link" key={doc['@id']}>
+                            <i className="icon icon-download" />&nbsp;
+                            <a data-bypass="true" href={attachmentHref} download={doc.attachment.download}>
```

Comment:
I know you just moved this code so this is an unrelated side note, but is there a reason we don't always skip js-based navigation for links with a `download` attribute, so that we could avoid also using `data-bypass`?

Reply:
Do you mean modifying the navigate method to make an exception for download links?

Reply:
I'm suggesting that be created as a new ticket. Similar to ENCD-1813 which did so based on `@@download` in the URL.